### PR TITLE
added the Algorand Example

### PR DESCRIPTION
**Pull request by Saviour1001**
Added the following functions to interact with Algorand 
- Get Algorand Keypair 
- Get Account
- Get Balance 
- Sign Message
- Sign and Send Transaction

### CODE REVIEW

File: web-core-sdk/algorand/react-algorand-core-example/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    setProvider(web3authProvider);
+  };
+
+  const getUserInfo = async () => {
+    if (!web3auth) {
+      console.log("web3auth not initialized yet");
+      return;
+    }
+    const user = await web3auth.getUserInfo();
+    console.log(user);
```
Comment (shahbaz17) [inline web-core-sdk/algorand/react-algorand-core-example/src/App.tsx:71]: Can you please change these `console.log` to `uiConsole`?

Comment (shahbaz17) [reply web-core-sdk/algorand/react-algorand-core-example/src/App.tsx:71]: Use it from https://github.com/Web3Auth/examples/blob/main/web-modal-sdk/multi-chain/src/App.tsx#L277

File: web-core-sdk/algorand/react-algorand-core-example/src/algorandRPC.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  }
+
+  getAlgorandKeyPair = async (): Promise<any> => {
+    const privateKey = (await this.provider.request({
+      method: "private_key",
+    })) as string;
+    var passphrase = algosdk.secretKeyToMnemonic(
+      Buffer.from(privateKey, "hex")
+    );
+    var account = algosdk.mnemonicToSecretKey(passphrase);
```
Comment (shahbaz17) [inline web-core-sdk/algorand/react-algorand-core-example/src/algorandRPC.ts:18]: This is `key-pair`, we shouldn’t write `account` for this. Please change everywhere too.

account is in `obj.addr` right?

File: web-core-sdk/algorand/react-algorand-core-example/src/App.tsx (hunk 1 above)
Comment (Saviour1001) [reply web-core-sdk/algorand/react-algorand-core-example/src/App.tsx:71]: Done !

File: web-core-sdk/algorand/react-algorand-core-example/src/algorandRPC.ts (hunk 2 above)
Comment (Saviour1001) [reply web-core-sdk/algorand/react-algorand-core-example/src/algorandRPC.ts]: Yeah, made the changes as per requested. The public key is ```obj.addr``` .

**Review by shahbaz17 — CHANGES_REQUESTED** [review]
@Saviour1001 

Styling is not good, can you update it to match styling from https://github.com/Web3Auth/examples/tree/main/web-modal-sdk/multi-chain?

Also, please add this Algorand example link to the [README of this project](https://github.com/Web3Auth/examples#readme).

**Comment by Saviour1001** [comment]
Added the Modal example for Algorand as well, followed the styling as suggested in both the examples. 
Made changes to readme.

**Review by shahbaz17 — APPROVED** [review]
LGTM

**Review by yashovardhan — APPROVED** [review]
I think it is good to merge, we will do all the linting and other changes iteratively.

**Pull request merged**